### PR TITLE
bpo-27741: Better wording for datetime.strptime()

### PR DESCRIPTION
**Pull request by augustogoulart**
https://bugs.python.org/issue27741

### CODE REVIEW

**Review by vstinner — APPROVED** [review]
LGTM.

@pganssle, @abalkin: Would you mind to double check?

**Comment by augustogoulart** [comment]
@pganssle I don't think the extra words in your suggestion would be a problem. My only concern is that we are describing something as **equivalent**, and pointing an **exception** just after.
Grammatically speaking, I really don't know if we can add an exception to something that is equivalent.

**Comment by taleinat** [comment]
I agree with @pganssle's suggestion here.

Minor suggestion: "but not `time.strptime`" -> "but are discarded by `time.strptime`", since `time.strptime` won't fail, but will silently ignore such input.

**Review by taleinat — APPROVED** [review]

**Comment by vstinner** [comment]
@augustogoulart: Can you please make the change suggested by @taleinat?

@pganssle: I would prefer to wait for your ACK on this PR before merging it. Would you mind to Approve it once it looks good to you?

**Comment by pganssle** [comment]
@vstinner Looks good to me, but @taleinat did have a sensible suggestion, so it might be best to implement that change.

**Comment by augustogoulart** [comment]
@taleinat @pganssle @vstinner changes commited.

File: Doc/library/datetime.rst
Comment (taleinat) [inline Doc/library/datetime.rst]:
```suggestion
which are supported in ``datetime.strptime`` but are discarded by ``time.strptime``.
```

File: Doc/library/datetime.rst
Comment (taleinat) [inline Doc/library/datetime.rst]:
```suggestion

```

**Review by taleinat — COMMENTED** [review]
IMO the last part of the sentence is unnecessary.

**Comment by augustogoulart** [comment]
I have made the requested changes; please review again.

**Comment by vstinner** [comment]
Is there anyone interested by this other strptime() doc issue? https://bugs.python.org/issue19376

**Comment by taleinat** [comment]
> Is there anyone interested by this other strptime() doc issue? https://bugs.python.org/issue19376

I've marked that as "easy"; expect a PR rather quickly!

**Comment by vstinner** [comment]
> I've marked that as "easy"; expect a PR rather quickly!

Datetime is not easy, but we will see :-) See how many iterations were needed just for this simple doc change ;-)

**Pull request merged**
